src/tree/types.ts

```diff
@@ -24,6 +24,14 @@ export enum FileType {
   CONFIG_FILE = 'CONFIG_FILE',
 }
 
+export enum Icon {
+  /* Added one space after emoji so that it doesn't stick together
+  with the next phrase or delete a character */
+  FOLDER = '📂 ',
```

Comment:
This should be done somewhere else, not here. This is just a Map, don’t have one function do two things.

Reply:
Added this to the function file instead